api/versioning/versioning.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package versioning;
```

Comment:
FWIW, I'm not sold on this file package or location; ideally we consolidate with the annotation package that @lizan is putting together for field renames. Consider this a placeholder for now.

Reply:
I'd prefer at least prefix this with envoy, say `envoy.api.annotations`? Or were you intended this to be in UDPA at the end so didn't put the prefix?

Reply:
Sure, let's put this in UDPA under `udpa.api.annotations`? If we can get agreement on the rest of this PR, I'll put out a PR for that.